packages/website/tests/index.spec.ts

```diff
@@ -1,7 +1,30 @@
 import AxeBuilder from '@axe-core/playwright';
-import { test } from '@playwright/test';
+import { expect, test } from '@playwright/test';
 
-test('Index', async ({ page }) => {
-  await page.goto('/');
-  await new AxeBuilder({ page }).analyze();
+test.describe('Website', () => {
+  test('Axe', async ({ page }) => {
+    await page.goto('/');
+    await new AxeBuilder({ page }).analyze();
+  });
+
+  test('should have no errors', async ({ page }) => {
+    const errorMessages = [] as string[];
+    page.on('console', msg =>
+      msg.type() === 'error'
+        ? errorMessages.push(`[${msg.type()}] ${msg.text()}`)
+        : null,
+    );
+    await page.goto('/');
+    expect(errorMessages.length).toBe(0);
```

Comment:
This will make things easier to debug:

```suggestion
    expect(errorMessages).toBe([);
```